crates/core_arch/src/powerpc/altivec.rs

```diff
@@ -688,10 +688,13 @@ mod sealed {
                 let addr = (b as *const u8).offset(a);
 
                 // Workaround ptr::copy_nonoverlapping not being inlined
-                unsafe extern "rust-intrinsic" {
+                    #[rustc_intrinsic]
+                    #[rustc_intrinsic_must_be_overridden]
```

Comment:
Note that the `rustc_intrinsic_must_be_overridden` attribute is being removed, please do not use it any more. The new new way to declare an intrinsic is:
```rust
#[rustc_intrinsic]
unsafe fn unreachable() -> !;
```